# Single-Cell RNA-seq Analysis (including HANDS-ON on ASAP) - 2025 CAS Course

Course given by [Vincent Gardeux ](https://people.epfl.ch/vincent.gardeux)<br/>
Course website: [https://advancedstudies.unibas.ch/courses/course/cas-personalized-molecular-oncology-191983](https://advancedstudies.unibas.ch/courses/course/cas-personalized-molecular-oncology-191983)<br/>

***

2025 Apr. 11th - UNIL

<br/>

In recent years, single-cell transcriptomics has become a widely used technology to study heterogeneous and dynamic biological systems. A large number of new tools and approaches have been developed for analyzing this new type of data.

The goal of this course is to provide theoretical and practical knowledge in the basic analysis of single-cell RNA-seq datasets. Specifically, using an online platform, ASAP ([asap.epfl.ch](asap.epfl.ch)), to make the entire analysis code-free, reproducible and shareable.

This course covers the standard analysis of a single-cell RNA-seq dataset using [ASAP](asap.epfl.ch). ASAP (Automated Single-cell Analysis Portal) is a web-based ([asap.epfl.ch](asap.epfl.ch)), collaborative portal aimed at democratizing single-cell omics data analyses and rendering it more accessible to researchers. ASAP does not require any installation and enables standardized analyses that can be run in minutes by any user without requiring significant computing power. The entire single-cell analysis pipeline is available in ASAP, allowing users to choose from a panel of tools, and guiding them through tutorials.

In the hands-on part of this course, we will use a toy dataset consisting of 7,000 cells from the ["Fly Cell Atlas"](https://flycellatlas.org/) repository. These cells all come from the **Body** part, which was dissociated as a whole and sequenced using 10x Genomics technology over 10 batches, mixing males and females. This set of 7,000 cells was carefully selected/filtered from the original **Body dataset** that consists of 96,926 cells. They belong to *a certain number* of differently annotated cell types. The goal of the hands-on session is to 1) Preprocess the dataset using a standard pipeline, 2) find out how many cell types there are in the data set, and 3) annotate the cell types using ASAP or any other literature/expert curated knowledge about cell-types/marker genes.

<br/>

# <img border="0" src="https://www.svgrepo.com/show/20800/event-date-and-time-symbol.svg" width="40" height="40" style="vertical-align:middle;"> Date

***

2025.04.11 - UNIL - Lausanne (Switzerland)

<br/>

# <img border="0" src="https://www.svgrepo.com/show/4199/placeholder-on-a-map.svg" width="40" height="40" style="vertical-align:middle;"> Location

***

This course will take place on site at Lausanne, University of Lausanne, campus UNIL-Sorge, building AmphipÃ´le.

<br/>

# <img border="0" src="https://www.svgrepo.com/show/158264/schedule.svg" width="40" height="40" style="vertical-align:middle;"> Programme

***

- **09:00 â€“ 10:30**:  Introduction to single-cell analysis using ASAP (Part 1). Introduction to single-cell genomics, and more particularly single-cell transcriptomics.
- **10:30 â€“ 10:45**:  Coffee break
- **10:45 â€“ 12:30**:  Introduction to single-cell analysis using ASAP (Part 2). Live example of how we can use ASAP to perform single-cell analysis on an example dataset.
- **12:30 â€“ 13:30**:  Lunch break
- **14:00 â€“ 15:30**:  Practical exercises (Part 1). Hands-on session on a single-cell dataset where participants will perform all steps of the single-cell analysis pipeline themselves, and annotate cell types.
- **15:30 â€“ 15:45**:  Coffee break
- **15:45 â€“ 18:00**:  Practical exercises (Part 2). Hands-on session part 2 and wrap-up.

Day 3 â€“ 11 April 2025 â€“ 10h15-20h30 â€“ University of Lausanne, AmphipÃ´le 318<br/>

**Emerging technologies**<br/>

- **10:15 â€“ 12:15**:  Large Language Models (Dr. Ana-Claudia Sima, SIB)
- **12:15 â€“ 13:00**:  Lunch break
- **13:00 â€“ 14:00**:  Glycomics (Dr. FrÃ©dÃ©rique Lisacek, Unige & SIB)
- **14:00 â€“ 15:00**:  RNA-seq analyses (Agnieszka Kraft, ETHZ) REMOTELY
- **15:00 â€“ 15:15**:  Break
- **15:15 â€“ 17:15**:  Single-cell-seq analyses, including HANDS-ON on ASAP (Dr.Vincent Gardeux, EPFL)
  - **15:15 â€“ 16:15**:  Introduction to single-cell transcriptomics (Lecture)
  - **16:15 â€“ 17:15**:  Toy project analysis using ASAP (Hands-on)
- **17:15 â€“ 17:30**:  Break
- **17:30 â€“ 19:00**:  Multi-omics data integration (Dr. Florence Mehl)
- **19:00 â€“ 20:30**:  ApÃ©ro offered to participants

<br/>

# <img border="0" src="https://www.svgrepo.com/show/410/list.svg" width="40" height="40" style="vertical-align:middle;"> Learning objectives

***

At the end of the course, the participants are expected to:
- run a single-cell RNA-seq analysis pipeline using ASAP
- choose the best tools and parameters for each analysis step
- carry out cell-type annotation and downstream analyses

**Pre-requisites - material & software**
- The participants are thus required to have their own computer with an Internet connection. ASAP is a fully online tool, so no other installation is needed.

<br/>

# <img border="0" src="https://www.svgrepo.com/show/38706/group-of-people.svg" width="40" height="40" style="vertical-align:middle;"> Staff

***

## Lecturer:

- __*Vincent Gardeux*__, Senior Scientist, EPFL, Lausanne, ðŸ‡¨ðŸ‡­ Switzerland

<br/>
<br/>

**Additional information**

For more information, please contact [vincent.gardeux@epfl.ch](mailto://vincent.gardeux@epfl.ch).
